Digipost/src/main/java/no/digipost/android/gui/metadata/ExternalLinkWebview.java

```diff
@@ -0,0 +1,233 @@
+/**
+ * Copyright (C) Posten Norge AS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package no.digipost.android.gui.metadata;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.app.DownloadManager;
+import android.content.*;
+import android.graphics.drawable.ColorDrawable;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Bundle;
+import android.os.Environment;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+import android.view.Window;
+import android.view.WindowManager;
+import android.webkit.*;
+import no.digipost.android.R;
+import no.digipost.android.utilities.DialogUtitities;
+import no.digipost.android.utilities.FileUtilities;
+import no.digipost.android.utilities.Permissions;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import static java.lang.String.format;
+
+public class ExternalLinkWebview extends AppCompatActivity{
+
+    private WebView webView;
+    private String fileName;
+    private String fileUrl;
+    private BroadcastReceiver onComplete;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_externallink_webview);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        final ActionBar actionBar = getSupportActionBar();
+        if (actionBar != null) {
```

Comment:
det brekker ikke om den er null og du fortsetter etter if-en?

Reply:
Nope, den vil alltid finnes der, siden den blir laget rett over, men siden den blir satt og hentet ut fra viewet så må if'n være der. 👍 Brekker ikke uansett, men får warning, så er for skille gode fra dårlige warnings.